NDTensors/test/blocksparse.jl

```diff
@@ -258,7 +270,7 @@ using GPUArraysCore: @allowscalar
       randn!(A)
       U, S, V = svd(A)
       @test @allowscalar isapprox(
-        norm(array(U) * array(S) * array(V)' - array(A)), 0.0; atol=1e-13
+        norm(array(U) * array(S) * array(V)' - array(A)), 0.0; atol=atol
```

Comment:
I think it is nicer to do:
```julia
      @test @allowscalar array(U) * array(S) * array(V)' ≈ array(A)) atol=atol
```

Reply:
This makes sense should I change this across the board?

Reply:
Sure, thanks!